javascript/lib/arrow.ts

```diff
@@ -0,0 +1,184 @@
+import { flatbuffers } from 'flatbuffers';
+import { org } from './Arrow_generated';
+var arrow = org.apache.arrow;
+import { vectorFromField, Vector } from './types';
+
+export function loadVectors(buf) {
+    var fileLength = buf.length, bb, footerLengthOffset, footerLength,
```

Comment:
This expects an Arrow file in the "random access" format (the other one currently available is the streaming format, see http://wesmckinney.com/blog/arrow-streaming-columnar/). It should be at least documented here that this is expected, better would be if we could include it in the name.